OPTIONS.md

```diff
@@ -149,8 +149,7 @@ Options:
   --no-module-parser-javascript-unknown-context-recursive                            Negative 'module-parser-javascript-unknown-context-recursive' option.
   --module-parser-javascript-unknown-context-reg-exp <value>                         Sets the regular expression when using the require function in a not statically analyse-able way.
   --module-parser-javascript-unknown-context-request <value>                         Sets the request when using the require function in a not statically analyse-able way.
-  --module-parser-javascript-url                                                     Enable/disable parsing of new URL() syntax.
-  --no-module-parser-javascript-url                                                  Negative 'module-parser-javascript-url' option.
+  --module-parser-javascript-url <value>                                             Enable/disable parsing of new URL() syntax.
```

Comment:
Very strange, why we remove `--no` value?

Reply:
I just ran `yarn update:docs` it automatically got synced with webpack latest version. 

It changed because now webpack have `smipleType: 'string'` for this option. 

https://github.com/webpack/webpack/blob/2a0536cf510768111a3a6dceeb14cb79b9f59273/test/__snapshots__/Cli.test.js.snap#L2700

https://github.com/webpack/webpack/commit/5d57777a5ea965493f262a8ffcb821d680eb66a4

Reply:
We need to improve our arg parser for such cases.

Reply:
Do we have test for this? Because it looks like breaking change

Reply:
No, we don't have tests for `module-parser-javascript-url` , I will add some 👍🏻